src/Application/Services/ReservationService.cs

```diff
@@ -1,14 +1,16 @@
 ﻿using Application.Dtos;
 using Application.Dtos.Reservation.Requests;
 using Application.Dtos.Reservation.Responses;
+using Application.Exceptions;
 using Domain.Entities;
-using Infrastructure.Repositories;
-using ResrautantLayer.Exceptions;
+using RestaurantLayer.Exceptions;
 using RestaurantLayer.Repositories;
 
```

Comment:
1. Get existing reservation by LINQ query
2. Do not set new datetimeoffset for mapping
3. Use paging on get all